### ISSUE TITLE
When using 'No images' - Background images still work

### ISSUE THREAD

**Issue by JayOram** [wttw/aboutmyemail#1]
When viewing the email as 'No Images' - any background images are still visible. 

I tested using this email - [BBC Planet Earth](https://parcel.io/e/2dea40aa-0961-41d2-9a8b-d0b877a1023b)

![Screenshot 2024-01-02 at 14 52 04](https://github.com/wttw/aboutmyemail/assets/10621565/477b5f77-bc5b-44f7-87ed-ff31c2c3dd56)

**Comment by wttw** [comment]
Yup. Currently I parse images out of CSS, but don't rewrite them so that they're intercepted. It's on the to-do-eventually list.
